### PR TITLE
Adds manuals with information about the new species to medbay

### PR DESCRIPTION
**Pull request by Super3222**
#### Intent of your Pull Request

Manuals link to:
https://wiki.yogstation.net/index.php?title=Preternis
https://wiki.yogstation.net/index.php?title=Phytosian

##### Changelog

:cl:
rscadd: Medbay starts with manuals helping them treat preternis and phytosian alike.
/:cl:

next stop is giving medical doctors access to cloning.

### CODE REVIEW

**Comment by Time-Green** [comment]
You sure this doesnt use /tg/'s wiki?

**Comment by Super3222** [comment]
Good point

File: code/game/objects/items/weapons/manuals.dm [hunk 1]
```diff
@@ -962,4 +962,26 @@
...
 	title = "Surgery for Dummies"
-	page_link = "Surgery"
\ No newline at end of file
+	page_link = "Surgery"
+
+/obj/item/weapon/book/manual/wiki/preternis
+	name = "Managing your Preternis"
+	icon_state = "bookEngineering"
+	author = "United Earth Government"
+	title = "Surgery for Dummies"
```
Comment (Cruix) [inline code/game/objects/items/weapons/manuals.dm]: title

**Comment by Super3222** [comment]
@Cruix @Time-Green This is ready to go. Wiki books also uses the configs wiki url, which I changed a while ago to Yogstation's.

**Pull request merged**